samples/snippets/src/test/java/com/example/storage/bucket/PrintBucketAclFilterByUserTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.storage.bucket;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.storage.Acl;
+import com.google.cloud.storage.Acl.Entity;
+import com.google.cloud.storage.Acl.Role;
+import com.google.cloud.storage.Acl.User;
+import com.google.cloud.storage.BucketInfo;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import com.google.cloud.storage.testing.RemoteStorageHelper;
+import com.google.cloud.testing.junit4.StdOutCaptureRule;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class PrintBucketAclFilterByUserTest {
+
+  @Rule public StdOutCaptureRule stdOut = new StdOutCaptureRule();
+  private static final String USER_EMAIL =
+      "google-cloud-java-tests@" + "java-docs-samples-tests.iam.gserviceaccount.com";
```

Comment:
We probably want to parameterize this as an environment variable, similar to what is done for the integration tests.

See https://github.com/googleapis/java-storage/blob/9266cefe2b522493cbf2e940ff59baf8c8a0c426/.kokoro/presubmit/integration.cfg#L30-L33 for how they get passed in. And https://github.com/googleapis/java-storage/blob/9266cefe2b522493cbf2e940ff59baf8c8a0c426/google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java#L2433-L2434 for how it's loaded/resolved.